cerebrium/development/serve.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Faster development with Serve
+description: Use the `cerebrium serve` command to rapidly iterate on your development
+---
+
+When you are developing a `cortex` deployment on **cerebrium**, waiting for a build to complete can be time-consuming. To speed up your development process, you can use the `cerebrium serve` command to rapidly iterate on your deployment.
+
+This allows you to run your deployment on a dedicated server, and see the results of your changes quickly without rebuilding the entire deployment.
+For simple changes to your deployment, you can see the results of your changes in seconds, rather than minutes.
+
+<Note>
+  This feature is currently in alpha and is available to all users. As such, you
+  may encounter bugs or limitations. We are actively working on improving the
+  experience and adding more features. If you have any feedback or suggestions,
+  we'd love to hear from you on discord or slack!
+</Note>
+
+## How it works
+
+When you run `cerebrium serve start`, the following happens:
+
+1. The `cerebrium` CLI uploads your deployment to a dedicated server.
+2. The server builds your deployment in the same way as `cerebrium deploy`.
+3. The server starts your deployment and waits for you to send in requests.
+4. If you make changes to your main.py or other code in your deployment, the server reloads your deployment and applies your changes without rebuilding the entire deployment.
+5. When you're done, you can stop the server by pressing `Ctrl+C` in the same terminal where you started the server.
+
+## Usage
+
+To start a served instance, first navigate to the root folder of your **cortex deployment**.
+
+Then, simply run the following command in your terminal:
+
+```bash
+cerebrium serve start
+```
+
+After running the `cerebrium serve start` command, you will see output in your terminal indicating that the server is starting up. This output will include a URL that you can use to access your deployment locally.
+
+Here are some additional commands that can be useful when working with `cerebrium serve`:
+
+### Stopping a served instance
+
+To stop a served instance, you can simply press `Ctrl+C` in the terminal where the server is running. This will gracefully shut down the server and terminate the served instance.
+
+If, for some reason, your session does not close gracefully, you can also use the following command to end a served instance:
+
+```bash
+cerebrium serve stop <<serve_id>>
+```
+
+### Testing inference on the served instance
+
+#### Using the built in CLI predict utility
+
+The goal of the `cerebrium serve` is to allow you to build and test your deployment as quickly as possible.
```

Comment:
Take the below out. A user should only use the local server. If they want to run a terminal command they can use curl

The goal of the `cerebrium serve` is to allow you to build and test your deployment as quickly as possible.
To help with this, we've added a `cerebrium serve predict` command that allows you to send requests to your served deployment without having to leave your ide or terminal.
To use the `cerebrium serve predict` command, first start a served instance using `cerebrium serve start` and then run `cerebrium serve predict` with the input data for your deployment as a json object.
For example:
```bash
cerebrium serve predict '{"prompt": "this is my input data"}'
```